.dockerfiles/.env

```diff
@@ -1,4 +1,5 @@
 MARKUS__REDIS__URL=redis://redis:6379/
+MARKUS__PYTHON=/app/venv/bin/python
```

Comment:
Maybe I'm missing something here, but is this environment variable being used anywhere?

Perhaps related: I agree it makes sense to remove `python` from the `settings.yml` file, but I want to make sure it's active by default in development (in docker), or at the very least that we update the Wiki to include instructions to add this to a `settings.local.yml` or something.